app/controllers/cart_controller.rb

```diff
@@ -16,4 +16,10 @@ def create
 
     redirect_to photos_path
   end
+
+  def index
+    @cart_items = Cart.last.shopping_cart_items.all
+    @cart = Cart.last
```

Comment:
It might be merge conflicts. This is the base code you started to work from. Any merge issues need to be handled with.